studio/src/mas-recently-updated.js

```diff
@@ -1,6 +1,7 @@
 import { html, LitElement, nothing } from 'lit';
 import Store from './store.js';
 import StoreController from './reactivity/store-controller.js';
+import { VARIANTS } from './editors/variant-picker.js';
```

Comment:
with this we would need to agree to not merge mas PRs before Milo PRs, but i guess its fine.
currently i did this mistake adding plans variant to MAS main branch before respective milo PR was merged, so plans cards will show up:
https://mwpw-167655--mas--adobecom.aem.live/studio.html#path=nala
![image](https://github.com/user-attachments/assets/f3de89b8-1bf9-4fc7-9a2a-ce37482dea91)

But we can announce new 'practise' and follow it from now on
Also if we only test in sandbox, people will be less concerned